docs/src/orchid/resources/wiki/guide/customize-defaults.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+Pebble comes with a rich set of built-in tags and filters that will help you render your templates into  websites and other documents with ease. However, imagine a more specific use-case where the templates are not entirely under your control.
+
+In these cases it might be advised to consider stripping-down Pebbles' built-in functionality that may otherwise introduce security-concers regarding the integrity and stability of your application.
+
+### Opt-Out using ExtensionCustomizer
+
+The `ExtensionCustomizer` base class can be used to gain access to the default functionality before it is loaded into Pebbles template engine. Overwrite methods to get hold on provided default-functionality and modify whatever should be available for the template engine.
+
+The following example removes the `ForTokenParser`, i.e. the ability to parse `{% for %}{{ ... }}{% endfor %}` constructs:
+
+```java
+class ExampleOptOuts extends ExntensionCustomizer {
```

Comment:
there's a typo in `ExntensionCustomizer`

Reply:
:facepalm: fixed it :)